e2e/tests/home/home-page.spec.ts

```diff
@@ -39,7 +39,7 @@ describe('Home page ui tests', () => {
       expect(cardName).toMatch(new RegExp(/^[a-zA-Z]+/));
 
       // check Workspace Action menu for listed actions
-      const ellipsis = await card.getEllipsis();
+      const ellipsis = card.getEllipsis();
```

Comment:
is this correct?

Reply:
yes, correct.  It's non-asynchronous method.

Reply:
ok